File: fjt.api/models/RFQLineItemAutoPricingStatus.js
const Sequelize = require('sequelize');

module.exports = (sequelize) => {
    const RFQLineItemAutoPricingStatus = sequelize.define('RFQLineItemAutoPricingStatus', {
        id: {
            allowNull: false,
            autoIncrement: true,
            primaryKey: true,
            type: Sequelize.INTEGER
        },
        rfqAssyID: {
            allowNull: false,
            type: Sequelize.INTEGER
        },
        consolidateID: {
            allowNull: false,
            type: Sequelize.INTEGER
        },
        pricingApiName: {
            type: Sequelize.STRING,
            allowNull: true
        },
        status: {
            type: Sequelize.INTEGER,
            allowNull: true
        },
        msg: {
            type: Sequelize.TEXT,
            allowNull: true
        },
        errorMsg: {
            type: Sequelize.TEXT,
            allowNull: true
        },
        userID: {
            type: Sequelize.INTEGER,
            allowNull: false
        },
        statusChangeDate: {
            type: Sequelize.DATE,
            allowNull: false
        },
        isPurchaseApi: {
            type: Sequelize.BOOLEAN,
            allowNull: true
        },
        pricingSupplierID: {
            type: Sequelize.INTEGER,
            allowNull: true
        }
    }, {
        paranoid: true,
        timestamps: false,
        tableName: 'rfq_lineitem_autopricingstatus'
    });

    RFQLineItemAutoPricingStatus.associate = () => {
        // RFQLineItemAutoPricingStatus.belongsTo(models.RFQConsolidatedMFGPNLineItem, {
        //    as: 'rfqConsolidatedMFGPNLineItem',
        //    foreignKey: 'consolidateID',
        // });
    };

    return RFQLineItemAutoPricingStatus;
};

File: fjt.pricingservice/MySqlDBModel/rfq_lineitem_autopricingstatus.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace fjt.pricingservice.MySqlDBModel
{
    [Table("rfq_lineitem_autopricingstatus")]
    public partial class rfq_lineitem_autopricingstatus
    {
        [Key]
        public int id { get; set; }
        public int rfqAssyID { get; set; }
        public int consolidateID { get; set; }
        public string pricingApiName { get; set; }
        public int? status { get; set; }
        public string msg { get; set; }
        public string errorMsg { get; set; }
        public int userID { get; set; }
        public DateTime statusChangeDate { get; set; }
        public bool isPurchaseApi { get; set; }
        public int? pricingSupplierID { get; set; }
        //public virtual rfq_consolidated_mfgpn_lineitem rfq_consolidated_mfgpn_lineitem { get; set; }

    }
}
